TicketSystem/TicketSystem/ViewModels/UserRoles/UserRoleEntity.cs

```diff
@@ -1,8 +1,7 @@
-﻿namespace TicketSystem.DAL.Entities
+﻿namespace TicketSystem.DAL.Entities;
+
+public class UserRoleVm
```

Comment:
Same, and rename file

Reply:
Also better to add not short view models, currently u have several cases when get vm missed